modular_skyrat/modules/officestuff/code/officestuff.dm

```diff
@@ -0,0 +1,18 @@
+/obj/structure/awaymissions/wildwest/masterclock
```

Comment:
why wild west structure?

Reply:
oh yeah, i originally planned to use that for my wild west rework, I was just too lazy to change it